contrib/kubernetes/flowlogs2metrics.conf.yaml

```diff
@@ -173,7 +173,7 @@ pipeline:
       - srcK8S_Namespace
       - srcK8S_Type
       operation: sum
-      recordkey: Bytes
+      recordkey: bytes
```

Comment:
Small bug -- using need to use `bytes` and not `Bytes` we convert this in the generic transform